.github/workflows/tests.yml

```diff
@@ -15,17 +15,17 @@ jobs:
         python-version: [3.7, 3.8]
```

Comment:
Could you add 3.9 and 3.10? It's some time ago, since we updated the python version and in the meantime numpy dropped the support for 3.7 and 3.8 (https://numpy.org/neps/nep-0029-deprecation_policy.html).

We can keep 3.7 and 3.8, since they are still working.